### PR TITLE
Rebeka - project MongoDB

### PR DESCRIPTION
**Pull request by rebkova**
https://project-mongo-db.herokuapp.com/

### CODE REVIEW

**Review by Bluepoisson — APPROVED** [review]
Good job Rebeka :) 
This is probably the shortest review ever, all I have left to say is: approved.

File: server.js [hunk 1]
```diff
@@ -28,10 +56,51 @@ app.use(cors())
 app.use(bodyParser.json())
```
[the remark this answers is not in the document]
Comment (rebkova) [reply server.js:57]: Thanks for that! I have added the code now.

File: server.js [hunk 2]
```diff
@@ -1,21 +1,49 @@
...
+  average_rating: Number,
+  isbn: String,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+//custom environmental variable
```
[the remark this answers is not in the document]
Comment (rebkova) [reply server.js:26]: I kept those comments for my better understanding of the new terms and also to understand what the lines of code were suppose to do.

**Review by KarolCislowski — APPROVED** [review]
Nice work 👍🏻

**Pull request closed without merging**